crowbar_framework/spec/models/proposal_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe Proposal do
   before do
-    Proposal.any_instance.stubs(:update_corresponding_proposal_object).returns(true)
+    allow_any_instance_of(Proposal).to receive(:update_corresponding_proposal_object).and_return(true)
```

Comment:
Metrics/LineLength: Line is too long. [102/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)